pkg/runtime/reconciler.go

```diff
@@ -202,6 +202,8 @@ func (r *resourceReconciler) reconcile(
 			return r.deleteResource(ctx, rm, res)
 		}
 
+		rlog := ackrtlog.FromContext(ctx)
+		rlog.Info("AWS resource will not be deleted - deletion policy set to retain")
```

Comment:
Is it possible to include a mention of which resource is being retained?  Is there a convenient resource name available in `res` that you can copy a bit above before deletion?

Reply:
This is automatically added into the logs when using the `rlog` object. I posted a comment underneath your last comment asking for logs with an example log message

Reply:
https://github.com/aws-controllers-k8s/runtime/pull/107/files#r1065230998